tools/rna_tools/rnasnp/rnasnp.xml

```diff
@@ -37,8 +37,8 @@
         --edist=$edist
         --boltzmannPreFactor=$boltzmannPreFactor
         --winsizeFold=$mode.winsizeFold
-        > $outfile
-
+        > '$outfile'
+        ; grep "SNP" '$outfile' > /dev/null 
```

Comment:
In this case this one should be used: https://docs.galaxyproject.org/en/master/dev/schema.html#tool-stdio-exit-code